src/Symfony/Component/HttpKernel/Tests/ControllerMetadata/ArgumentMetadataFactoryTest.php

```diff
@@ -121,7 +121,7 @@ public function testNullableTypesSignature()
             new ArgumentMetadata('foo', 'string', false, false, null, true),
             new ArgumentMetadata('bar', \stdClass::class, false, false, null, true),
             new ArgumentMetadata('baz', 'string', false, true, 'value', true),
-            new ArgumentMetadata('mandatory', null, false, false, null, true),
+            new ArgumentMetadata('mandatory', null, false, false, null, false),
```

Comment:
This test was flawed originally, we should not pass null to a mandatory argument (thus isNullable should always have been false here)